homeassistant/components/lektrico/coordinator.py

```diff
@@ -0,0 +1,52 @@
+"""Coordinator for the Lektrico Charging Station integration."""
+
+from __future__ import annotations
+
+from datetime import timedelta
+from typing import Any
+
+from lektricowifi import DeviceConnectionError, lektricowifi
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    ATTR_HW_VERSION,
+    ATTR_SERIAL_NUMBER,
+    CONF_HOST,
+    CONF_TYPE,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.httpx_client import get_async_client
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import LOGGER
+
+SCAN_INTERVAL = timedelta(seconds=10)
+
+
+class LektricoDeviceDataUpdateCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class LektricoDeviceDataUpdateCoordinator(DataUpdateCoordinator[..]):
```
Should match return value of `_async_update_data`

Reply:
I made the change, thank you.